Library/Formula/pcap_dnsproxy.rb

```diff
@@ -0,0 +1,41 @@
+class PcapDnsproxy < Formula
+  desc "A powerful DNS proxy designed to anti DNS spoofing"
+  homepage "https://github.com/chengr28/Pcap_DNSProxy"
+  url "https://github.com/chengr28/Pcap_DNSProxy/archive/v0.4.2.tar.gz"
+  sha256 "0737b16283e84bc94f63066f448ba84416141b8d5a86b5e01f1f97164d9d44f2"
+  head "https://github.com/chengr28/Pcap_DNSProxy.git"
+
+  depends_on :xcode => :build
+  depends_on "libsodium" => :build
+
+  def install
+    ln_s "#{HOMEBREW_PREFIX}/lib/libsodium.a", "#{buildpath}/Source/LibSodium/LibSodium_Mac.a"
+    xcodebuild "-project", "./Source/KeyPairGenerator.xcodeproj", "-target", "KeyPairGenerator", "-configuration", "Release", "SYMROOT=build"
+    xcodebuild "-project", "./Source/Pcap_DNSProxy.xcodeproj", "-target", "Pcap_DNSProxy", "-configuration", "Release", "SYMROOT=build"
+
+    bin.install "Source/build/Release/KeyPairGenerator"
+    bin.install "Source/build/Release/Pcap_DNSProxy"
+    (etc/"pcap_DNSproxy").install Dir["Source/ExampleConfig/*.{ini,txt,plist}"]
+  end
+
+  def caveats
+    <<-EOS.undent
```

Comment:
We don't need such caveats. Instead you should use [`plist` method](https://github.com/Homebrew/homebrew/blob/master/Library/Contributions/example-formula.rb#L448)